plugin/pkg/admission/runtimeclass/admission.go

```diff
@@ -136,28 +137,17 @@ func (r *RuntimeClass) prepareObjects(attributes admission.Attributes) (pod *api
 	}
```

Comment:
Mind fixing the comment on line 130 while you're here?

Reply:
Or, see the following PR (feel free to cherry-pick):
https://github.com/kubernetes/kubernetes/pull/79621